io.github.Nheko-Reborn.Nheko.json

```diff
@@ -0,0 +1,199 @@
+{
+  "id": "io.github.Nheko-Reborn.Nheko",
+  "command": "nheko",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.gcc8"
+  ],
+  "rename-icon": "nheko",
+  "rename-desktop-file": "nheko.desktop",
+  "rename-appdata-file": "nheko.appdata.xml",
+  "build-options": {
+    "env": {
```

Comment:
since you don't need the GCC8 extension you shouldn't need any of these build time env vars. It should 'just work'

Reply:
Got it.  I removed the entire build-options section.